index.js

```diff
@@ -617,6 +617,15 @@
     function(x) { return RegExp_._test(x) && !x.global; }
   );
 
+  //# NonNegativeInteger :: Type
+  //.
+  //. Type comprising `0`, `-0`, and every [`PositiveInteger`][] value.
+  //. Also known as the set of natural numbers under ISO 80000-2:2009.
+  var NonNegativeInteger = NullaryTypeWithUrl(
+    'sanctuary-def/NonNegativeInteger',
+    function(x) { return PositiveInteger._test(x) || x === 0; }
```

Comment:
I'm good with that. I had actually changed the test to
```JavaScript
function(x) { return Integer._test(x) && x >= 0; }
```
earlier, then waffled, thought about asking you and finally changed it back.

I guess it makes more sense to choose the inclusive type and add a filter than it does to choose the exclusive type and add an exception to the rule.